src/components/Form/Search.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import T from 'prop-types'
+import { randomId } from '../../utils/'
+
+import SearchWrapper from './SearchWrapper'
+import SearchSubmit from './SearchSubmit'
+import SearchInput from './SearchInput'
+
+class Search extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: ''
+    }
+  }
+
```

Comment:
`isControlled = () => !!this.props.value`

Reply:
done